falk/vm/__init__.py

```diff
@@ -61,21 +61,19 @@ def __init__(self, machine_id, cfg, cfgpath):
             if value:
                 # ssh key loading:
                 if key == "ssh_known_host_key":
-                    # it's either the key directly or a file
-                    if value.startswith("ssh-"):
-                        # it's directly stored
-                        self.ssh_known_host_key = value
-                    else:
-                        # it's given as path to public key storage file
-                        path = Path(os.path.expanduser(value))
-
+                    # either the key is a file, or a line from the known hosts file.
+                    path = Path(os.path.expanduser(value))
+                    print(path)
+                    if path.is_file():
```

Comment:
we should think about some more feedback, otherwise a typo in the filename will lead to a non-working setup as the filename is then treated as key content.
suggestion: we'll write `ssh_known_host_key = keycontent` or  `ssh_known_host_key_file = filename` (and add both of them to the example documentation).

Reply:
Good idea, that also makes sense when ssh_known_host_key is used again by the Kevin process. It doesn't have access to the filesystem it came from, and so is relying upon falk to read it contents of it.